entropylab/api/param_store.py

```diff
@@ -73,6 +71,21 @@ def list_commits(self, label):
         """
         pass
 
+    @abstractmethod
+    def list_values(self, key: str) -> List[Tuple[Any, datetime, str, str]]:
```

Comment:
@urig can you update the type hint here as well?

Reply:
Tx!

Just a thought: is there a tool that can warn about mistakes like this?